packages/ember-flight-icons/src/components/flight-icon.ts

```diff
@@ -6,17 +6,49 @@
 import Component from '@glimmer/component';
 import { guidFor } from '@ember/object/internals';
 import { assert } from '@ember/debug';
-
 import { iconNames } from '@hashicorp/flight-icons/svg';
 
-export default class FlightIconComponent extends Component {
-  constructor() {
-    super(...arguments);
+import type { IconName } from '@hashicorp/flight-icons/svg';
+
+export interface FlightIconSignature {
+  Args: {
+    /**
+     * The color of the icon. Defaults to `currentColor`.
+     */
+    color?: string;
+    /**
+     * The name of the icon to display.
+     */
+    name?: IconName;
+    /**
+     * The size of the icon. Defaults to `16`.
+     */
+    size?: '16' | '24';
+    /**
+     * The title of the icon. If provided, the icon will be given an `aria-labelledby` attribute.
+     */
+    title?: string;
+    /**
+     * Whether the icon should be displayed as an inline block. Defaults to `true`.
+     */
+    isInlineBlock?: boolean;
+    /**
+     * Whether the icon should be stretched to fill its container. Defaults to `false`.
+     */
+    stretched?: boolean;
+  };
+  Element: SVGElement;
+}
+
+export default class FlightIcon extends Component<FlightIconSignature> {
+  constructor(owner: unknown, args: FlightIconSignature['Args']) {
+    super(owner, args);
+
     if (!this.args.name) {
       assert('Please provide to <FlightIcon> a value for @name');
     } else if (!iconNames.includes(this.args.name)) {
       assert(
-        `The icon @name "${this.args.name}" provided to <FlightIcon> is not correct. Please verify it exists on https://helios.hashicorp.design/icons/library`
+        `The icon @name "${this.args.name}" provided to <FlightIcon> is not correct. Please verify it exists on https://helios.hashicorp.design/icons/library`,
```

Comment:
why this extra comma at the end of the line?
<img width="653" alt="modified" src="https://github.com/hashicorp/design-system/assets/686239/2eb860ec-7c20-4c29-9aa9-5042fcf667fc">

Reply:
in the newer prettier version dangling comma is a default now

Reply:
left a comment about it. I would prefer if updates like this are done systematically across the monorepo, to avoid that different parts behave in different ways.